### ISSUE TITLE
Decoder: how to composite the next rnn_input?

### ISSUE THREAD

**Issue by jinyadong-dev** [jiangyuan2li/Deep-Spatio-Temporal#8]
I'm not sure whether you will revisit this repo or not, but I really think it's a perfect idea behind your paper, I keen to reproduce it as soon as possible, when I read through your model , I raise a question for your consideration, original code is as following:
            rnn_input = y_[i, :, 1:]
            rnn_input = torch.cat((rnn_input, output), dim=1)
            rnn_input = rnn_input.unsqueeze(0)

But according to the Decoder logic,  it maybe better to swipe it as:
            rnn_input = torch.cat((output,rnn_input), dim=1)
Otherwise, the sequence will not align with the first row of data.

Hope to hear from you.
